From a8ed9b8695065d37d45efc446efbde5654f9f7c2 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliangtang@163.com>
Date: Thu, 10 Mar 2016 00:18:57 -0500
Subject: ext4: drop unneeded BUFFER_TRACE in ext4_delete_inline_entry()

BUFFER_TRACE info "call ext4_handle_dirty_metadata" doesn't match the
code, so drop it.

Signed-off-by: Geliang Tang <geliangtang@163.com>
Signed-off-by: Theodore Ts'o <tytso@mit.edu>
---
 fs/ext4/inline.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fs/ext4/inline.c b/fs/ext4/inline.c
index 36d8cc9..7cbdd375 100644
--- a/fs/ext4/inline.c
+++ b/fs/ext4/inline.c
@@ -1697,7 +1697,6 @@ int ext4_delete_inline_entry(handle_t *handle,
 	if (err)
 		goto out;
 
-	BUFFER_TRACE(bh, "call ext4_handle_dirty_metadata");
 	err = ext4_mark_inode_dirty(handle, dir);
 	if (unlikely(err))
 		goto out;
-- 
cgit v0.12

